pkg/kubectl/cmd/rollout/rollout_history.go

```diff
@@ -49,6 +49,9 @@ var (
 func NewCmdRolloutHistory(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &HistoryOptions{}
 
+	validArgs := []string{"deployment"}
```

Comment:
> @deads2k definitely. As a short-term solution could this be an argument in NewCmdRolloutHistory?

I can't say that I'm a fan of that.  I think we should let this merge and you can work it out with @kargakis since he's trying to integrate these with openshift.